health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/service/ExerciseLogger.kt

```diff
@@ -2,12 +2,19 @@ package com.example.exercisesamplecompose.service
 
 import android.util.Log
 
+private const val TAG = "ExerciseSample"
+
 interface ExerciseLogger {
+    fun error(message: String)
     fun log(message: String)
 }
 
 class AndroidLogExerciseLogger : ExerciseLogger {
+    override fun error(message: String) {
```

Comment:
Can you take an optional Throwable here?

Reply:
Done.